pkg/provider/github/app/token.go

```diff
@@ -50,31 +47,27 @@ func (ip *Install) GetAndUpdateInstallationID(ctx context.Context) (string, stri
 		return "", "", 0, err
 	}
 
-	installationURL := *ip.ghClient.APIURL + keys.InstallationURL
```

Comment:
I see  `keys.InstallationURL` is not used anywhere can you remove that?

Reply:
Removed.